ContactManger/src/main/java/Contact.java

```diff
@@ -0,0 +1,36 @@
+public class Contact {
+
+    private String name;
+    private String email;
+    private String phone;
+
+    public Contact(String name, String email, String phone){
```

Comment:
@anamariaTapia Don't forget to add comments in all the methods, even if the method is simple this will help you to have good practices